ui/src/components/widgets/FormPopup/FormPopup.tsx

```diff
@@ -51,19 +51,18 @@ export function FormPopup(props: FormPopupProps) {
                     confirm: 'ok'
```

Comment:
You could have passed closeViewPopup to the onSuccessAction parameter, then you wouldn't need to create a separate epic for closing it